### PR TITLE
feat(fog view router): Adds deployments config

### PR DESCRIPTION
**Pull request by aweis89**
### Motivation

- Adds new fog view router binary to the fog view image
- Moves prior fog-view ingresses to point to the new fog view router('s SVC)
- Creates new headless SVC for fog-view stores, used in the `--client-responder-id` for rerouting to individual replicas
- Moves fog-view deployment to new CRD `FogStoreRollout` that's managed by the new [fog-operator ](https://github.com/mobilecoinofficial/fog-operator)

### Future Work

- GRPC health check is commented out in the router manifests, currently the rust code is being inspected/modified to support that
- The list of shards is currently statically defined in the helm values. However the operator and CR architecture should enable to us to generate that list dynamically in-response to block chain size or other factors

[Soundtrack of this PR]()

### CODE REVIEW

File: mobilecoind/strategies/requirements.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ grpcio==1.44.0
 grpcio-tools==1.44.0
 idna==3.3
 ipdb==0.12.2
-ipython==8.10.0
```
Comment (jcape) [inline mobilecoind/strategies/requirements.txt:13]: Looks like one of these branches is out-of-date with master...

**Review by jcape — CHANGES_REQUESTED** [review]

File: mobilecoind/strategies/requirements.txt (hunk 1 above)
Comment (aweis89) [reply mobilecoind/strategies/requirements.txt]: I think this actually causes a [failure](https://github.com/mobilecoinfoundation/mobilecoin/actions/runs/4255320061/jobs/7402819293) in the build process

File: .internal-ci/helm/fog-services/templates/fog-view-rollout-crd.yaml [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+          spec:
+            # Try to balance pods across zones
+            volumes:
+            - emptyDir: {}
+              name: aesm-socket-dir
+            - name: supervisor-conf
+              projected:
+                defaultMode: 420
+                sources:
+                - configMap:
```
Comment (aweis89) [inline .internal-ci/helm/fog-services/templates/fog-view-rollout-crd.yaml:37]: Add `mc-admin-http-gateway` supervisord` configmap

**Review by jgreat — APPROVED** [review]
Approved based on changes discussed with walk through.  Additional enhancements can be added incrementally after this main chunk of work is meged in.

File: mobilecoind/strategies/requirements.txt (hunk 1 above)
Comment (jcape) [reply mobilecoind/strategies/requirements.txt:13]: Please open an issue so we can resolve this, don't revert dependency updates.

Comment (aweis89) [reply mobilecoind/strategies/requirements.txt:13]: https://github.com/mobilecoinfoundation/mobilecoin/issues/3207

**Comment by jcape** [comment]
Closing this, if necessary re-open.

**Pull request closed without merging**